can_tools/__init__.py

```diff
@@ -38,13 +38,42 @@ def all_subclasses(cls):
     )
 
 
-def scrapers_for_flow(cls):
-
+def fetch_all_unblocked_scrapers(cls):
     blocked = [PennsylvaniaVaccineDemographics, TableauDashboard, CDCTestingBase]
```

Comment:
Good flag--yeah these are blocked explicitly because they're base/parent classes that, while they aren't abstract classes, aren't functional scrapers. I think they're blocked here b/c if a block flag (`autodag = False`) were to be added to these classes, it would need to be overwritten in all of their subclasses in order for those classes to be included. 

It is a little clunky, especially how we block/exclude different scrapers in different places, but for now, I think I might just add a comment to clarify this if that's okay

Reply:
Cool, sounds good!